src/Common/Commands.Common/AzurePSCmdlet.cs

```diff
@@ -271,6 +273,10 @@ protected override void BeginProcessing()
             SetupDebuggingTraces();
             SetupHttpClientPipeline();
             base.BeginProcessing();
+
+            //Now see if the cmdlet has any Breaking change attributes on it and process them if it does
+            //This will print any breaking change attribute messages that are applied to the cmdlet
+            BreakingChangeAttributeHelper.processCustomAttributesAtRuntime(this.GetType(), new List<String>(this.MyInvocation.BoundParameters.Keys));
```

Comment:
To keep from breaking unit tests, you may need to put in some null checks here, as in:
```this.MyInvocation?.BoundParameters?.Keys```

Also, method names should be ```PascalCase``` instead of ```camelCase```

Reply:
done